httpclient/src/main/java/org/apache/http/client/config/RequestConfig.java

```diff
@@ -295,11 +293,11 @@ public int getConnectTimeout() {
 
     /**
      * Defines the socket timeout ({@code SO_TIMEOUT}) in milliseconds,
-     * which is the timeout for waiting for data  or, put differently,
+     * which is the timeout for waiting for data or, put differently,
      * a maximum period inactivity between two consecutive data packets).
      * <p>
      * A timeout value of zero is interpreted as an infinite timeout.
-     * A negative value is interpreted as undefined (system default).
+     * A negative value is interpreted as undefined.
```

Comment:
@bekker Thank you for cleaning up the javadocs but why do you want to remove `(system default)` from socket timeout description and leave it everywhere else?

Reply:
I was not so sure about other timeouts. Would it be better to make it same for other timeouts?

Reply:
I think we should keep the docs as is or expand on it, as opposed to removing information.

Reply:
@garydgregory You're right. I should add more info, not removing it.

But I'm not sure about meaning of 'undefined' here - is it 'undefined behavior'? Then there should be a warning about unexpected results.